src/components/Badge.tsx

```diff
@@ -0,0 +1,27 @@
+import CornertLogo from '@/components/CornerLogo'
+import DerivativeName from '@/components/DerivativeName'
+import QRCodeWrapper from '@/components/QRCodeWrapper'
+import React from 'react'
```

Comment:
Do you need it here?

Reply:
I changed tsconfig in a right way, so we don't need it anymore. Before that changes, it wasn't able to compile it without react import. Ty for catching this.